src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useChainflipStreamingProgress.tsx

```diff
@@ -0,0 +1,169 @@
+import type { TradeQuote, TradeQuoteStep } from '@shapeshiftoss/swapper'
+// TODO: Is this import allowed?
```

Comment:
It's correct and importing deeply nested modules from swapper using dist is in fact the only way as it stands unfortunately!